configure

```diff
@@ -409,6 +409,7 @@ export cc verbose
 # Determine the C compiler family (GCC, Clang, etc)
 
 ccfamily=`$cc -E cckind.c | grep '^[a-z]' | tr -s ' ' '-'`
+cc_has_debug_prefix_map=`$cc -E cc_has_debug_prefix_map.c | grep '^[a-z]'`
```

Comment:
What about just passing `-fdebug-prefix-map` and seeing if the compiler fails or not?

Reply:
Well, I reused the logic from `config/auto-aux/cckind.c`.
Actually, a third possibly would be to do all the work
in the aforementioned file.
I don't have a strong opinion.